client/signup/config/steps.js

```diff
@@ -87,5 +88,17 @@ module.exports = {
 		},
 		dependencies: [ 'siteSlug' ],
 		providesDependencies: [ 'theme', 'images' ]
+	},
+
+	'jetpack-user': {
+		stepName: 'jetpack-user',
+		apiRequestFunction: stepActions.createAccount,
+		providesToken: true,
+		props: {
+			headerText: i18n.translate( 'Create an account for Jetpack' ),
```

Comment:
This seems to be causing the tests to fail. You might need to mock i18n in the tests... Let me know if you need help with this.